src/components/global/RenderIf.jsx

```diff
@@ -0,0 +1,3 @@
+export default function RenderIf({ children, isTrue }) {
+	return isTrue ? children : null;
+}
```

Comment:
You dont need a component to "just" check simple conditions. Wherever you use this component, just ask for the condition itself:

`{isSomeState && <YourComponent />}`